lib/experimental/block-comments.php

```diff
@@ -113,3 +110,82 @@ function gutenberg_filter_comment_count_query_exclude_block_comments( $query ) {
 	return $query;
 }
 add_filter( 'query', 'gutenberg_filter_comment_count_query_exclude_block_comments' );
+
+if ( ! class_exists( 'WP_List_Table' ) ) {
+	require_once ABSPATH . 'wp-admin/includes/class-wp-list-table.php';
+}
+
+if ( ! class_exists( 'WP_Posts_List_Table' ) ) {
+		require_once ABSPATH . 'wp-admin/includes/class-wp-posts-list-table.php';
+}
+
+// Include class-custom-wp-posts-list-table.php
+require_once __DIR__ . '/class-custom-wp-posts-list-table.php';
+
+/**
+ * Use the custom posts list table class.
+ *
+ * @param string $class_name The original class name.
+ * @return string The modified class name.
+ */
+function use_custom_posts_list_table( $class_name ) {
+	if ( 'WP_Posts_List_Table' === $class_name  ) {
+		return 'Custom_WP_Posts_List_Table';
+	}
+	return $class_name;
+}
+add_filter( 'wp_list_table_class_name', 'use_custom_posts_list_table' );
+
+/**
+ * Add some label styles on the post list table.
+ */
+function gutenberg_add_open_discussion_label_styles() {
+	$screen = get_current_screen();
+	if ( 'edit-post' === $screen->id || 'edit-page' === $screen->id ) {
+		?>
+	<style>
```

Comment:
```suggestion
	<style>
```

In core, these styles will probably be part of the `list-tables.css` stylesheet, which is loaded on all admin screens. With that in mind, we might want to remove this conditional statement.

Reply:
Ok, sure!

Reply:
removed and made style names more specific to avoid conflicts in 9bb41beae8117852eb72079655f87cb0d3e5adec